_posts/2021-03-29-changelog-144.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #144"
+date: 2021-03-29 17:00:00 +0300
+---
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6799 -->
+* Complete dependencies by subwords in `Cargo.toml` when using experimental local crates index. This feature is disabled by default
```

Comment:
Let's reorder items a little bit.
I suggest the following order: import aliases in `Move` refactoring, const generics, `Complete dependencies` and other in current order

Reply:
I like this idea. Actually, I put this item on the top because of an attractive gif. But now we have another gif about `Move` refactoring 👌